tests/playbooks/tests_bridge_cloned_mac.yml

```diff
@@ -3,7 +3,7 @@
 - name: Test configuring bridges
   hosts: all
   tasks:
-    - name: define vars
+    - name: Define vars
```

Comment:
Using upper-case for names is another candidate for an individual commit to be merged.